src/main/java/uk/gov/hmcts/reform/feature/security/CustomAccessDeniedHandler.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.feature.security;
+
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.web.access.AccessDeniedHandler;
+import org.springframework.util.StreamUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static org.springframework.http.MediaType.TEXT_HTML_VALUE;
+
+public class CustomAccessDeniedHandler implements AccessDeniedHandler {
```

Comment:
Could we please add a test case to cover Access denied scenario ?

Reply:
Just a tic.. and it's there :)